lib/floe/cli.rb

```diff
@@ -82,5 +90,22 @@ def parse_options!(args)
 
       return workflows_inputs, opts
     end
+
+    def create_workflow(workflow, context_payload, input, credentials)
+      context = Floe::Workflow::Context.new(context_payload, :input => input, :credentials => credentials)
+      Floe::Workflow.load(workflow, context)
+    end
+
+    def create_loggers(workflows, segment_output)
+      if workflows.size == 1 || !segment_output
+        # no extra work necessary
+        {}
+      else
+        workflows.each_with_object({}) do |workflow, h|
+          workflow.context.logger = Logger.new(output = StringIO.new)
```

Comment:
Can you explain this string io part?  I don't understand this bit.  Is this meant to be like a NullLogger pattern (if so we should use an actual NullLogger like we do in other gems)?